AndroidIDPTemplate/app/src/com/salesforce/samples/salesforceandroididptemplateapp/MainApplication.kt

```diff
@@ -49,13 +49,6 @@ class MainApplication : Application() {
         val mobileSDKSampleAppCallbackUrl = "testsfdc:///mobilesdk/detect/oauth/done"
 
         MobileSyncSDKManager.getInstance().setAllowedSPApps(listOf(
-            SPConfig(
```

Comment:
For more details on removing this, see 👉🏻 https://github.com/forcedotcom/SalesforceMobileSDK-Android/pull/2482#issuecomment-1789723719